libraries/standard/mqtt/cbmc/proofs/MQTT_SerializeAck/Makefile

```diff
@@ -0,0 +1,17 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=MQTT_SerializeAck_harness
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROOF_SOURCES += $(PROOFDIR)/../../sources/mqtt_cbmc_state.c
```

Comment:
I generally think it would be better to give full, normalized path names without the ..

There is no engineering reason for this other than a fear in the back of my mind that some analysis tool in the future will assume normalized paths...

Reply:
ok

Reply:
So I will update this to $(SRCDIR)/libraries/standard/mqtt/cbmc/sources/mqtt_cbmc_state.c